ZenPacks/zenoss/ZenPackLib/lib/params/EventClassSpecParams.py

```diff
@@ -1,43 +1,43 @@
 ##############################################################################
 #
-# Copyright (C) Zenoss, Inc. 2016-2017, all rights reserved.
+# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
 #
 # This content is made available according to terms specified in
 # License.zenoss under the directory where your Zenoss product is installed.
 #
 ##############################################################################
-
-from .OrganizerSpecParams import OrganizerSpecParams
+from ..base.types import multiline
+from .SpecParams import SpecParams
 from .EventClassMappingSpecParams import EventClassMappingSpecParams
 from ..spec.EventClassSpec import EventClassSpec
 
 
-class EventClassSpecParams(OrganizerSpecParams, EventClassSpec):
+class EventClassSpecParams(SpecParams, EventClassSpec):
```

Comment:
Why is EventClassParams not an OrganizerSpecParams anymore? That seems like the natural thing since EventClass is an organizer.